sasmodels/models/bcc_paracrystal.py

```diff
@@ -64,19 +64,23 @@
     \text{Indices} & (110) &    (200) & (211)    & (220)    & (310)    \\
     \end{array}
 
-**NB**: The calculation of $Z(q)$ is a double numerical integral that must
-be carried out with a high density of points to properly capture the sharp
-peaks of the paracrystalline scattering. So be warned that the calculation
-is SLOW. Go get some coffee. Fitting of any experimental data must be
-resolution smeared for any meaningful fit. This makes a triple integral.
-Very, very slow. Go get lunch!
-
+.. note::
+
+  The calculation of $Z(q)$ is a double numerical integral that
+  must be carried out with a high density of points to properly capture
+  the sharp peaks of the paracrystalline scattering.     
+  So be warned that the calculation is slow. Fitting of any experimental data 
+  must be resolution smeared for any meaningful fit. This makes a triple integral
+  which may be very slow.
+  
 This example dataset is produced using 200 data points,
 *qmin* = 0.001 |Ang^-1|, *qmax* = 0.1 |Ang^-1| and the above default values.
 
 The 2D (Anisotropic model) is based on the reference below where $I(q)$ is
 approximated for 1d scattering. Thus the scattering pattern for 2D may not
-be accurate.
+be accurate, particularly at low $q$. For general details of the calculation and angular 
+dispersions for oriented particles see :ref:`orientation` .
+Note that we are not responsible for any incorrectness of the 2D model computation.
```

Comment:
Rather than "we are not responsible for any incorrectness", how about specifying what we did to check the model, or perhaps "the 2D model equations have not been verified.".  The overall BSD license that Andrew threw on in April already disclaims any responsibility.